text_generation/causal_lm/cpp/continuous_batching/library/src/continuous_batching_pipeline.cpp

```diff
@@ -55,18 +56,18 @@ class ContinuousBatchingPipeline::Impl {
     }
 
 public:
-    Impl(const std::string& models_path, const SchedulerConfig& scheduler_config) {
+    Impl(const std::string& models_path, const SchedulerConfig& scheduler_config, const std::string device = "CPU", const ov::AnyMap& plugin_config = {}) {
         ov::Core core;
         m_tokenizer = std::make_shared<Tokenizer>(models_path);
 
         // The model can be compiled for GPU as well
         std::shared_ptr<ov::Model> model = core.read_model(models_path + "/openvino_model.xml");
 
-        const std::string device = "CPU";
         DeviceConfig device_config(core, scheduler_config, device);
 
         apply_paged_attention_transformations(model, device_config);
-        ov::InferRequest infer_request = core.compile_model(model, device_config.get_device(), ov::enable_profiling(true)).create_infer_request();
```

Comment:
can we enable profiling in tput app? otherwise, current stat will be reported as empty after app is finished